mino/minows/mod.go

```diff
@@ -17,40 +20,43 @@ import (
 )
 
 // Minows implements mino.Mino
-// TODO unit tests
 type minows struct {
-	myAddr   address
-	segments []string
-	host     host.Host
-	rpcs     map[string]rpc
-	context  serde.Context
+	logger zerolog.Logger
+
+	myAddr    address
+	namespace []string
+	host      host.Host
+	context   serde.Context
+	rpcs      map[string]any
 }
 
-// NewMinows
-// listen: local listening address in multiaddress format, e.g. /ip4/0.0.0.0/tcp/80
-// public: public dial-able address in multiaddress format, e.g. /dns4/p2p-1.c4dt.dela.org/tcp/443/wss
-// identity: Peer ID (multi-hash of public key generated by libp2p)
-func NewMinows(listen, public ma.Multiaddr, privKey crypto.PrivKey) (*minows, error) {
-	id, err := peer.IDFromPrivateKey(privKey)
+// newMinows
```

Comment:
There should be a phrase at the beginning, e.g.,

```
// newMinows starts a new mino service using libp2p to connect over websockets.
```

Reply:
generated doc now